cardea/data_loader/entityset_loader.py

```diff
@@ -0,0 +1,71 @@
+from glob import glob
+
+import pandas as pd
+
+import featuretools as ft
+from cardea.data_loader import DataLoader
+
+
+class EntitySetLoader(DataLoader):
+
+    def create_entity(self, object, entity_set):
+        """ this method receives FHIR objects and creates featuretools entities and adds them to the
+        entityset
+        """
+
+        df = object.get_dataframe()
+
+        # get ID if exists
+        if 'identifier' in df.columns:
+            id = 'identifier'
+        elif 'id' in df.columns:
+            id = 'id'
+        else:
```

Comment:
Is there any chance that we add additional identifier column names in the future? If so, I recommend we place `object_id` in an `elif` and add `raise LookupError('{} is missing an identifier column'` in the `else` statement.  This adds extra protection.